maskrcnn_benchmark/layers/roi_align.py

```diff
@@ -50,11 +52,15 @@ def backward(ctx, grad_output):
 class ROIAlign(nn.Module):
     def __init__(self, output_size, spatial_scale, sampling_ratio):
         super(ROIAlign, self).__init__()
-        self.output_size = output_size
+        self.output_size = _pair(output_size)
         self.spatial_scale = spatial_scale
         self.sampling_ratio = sampling_ratio
 
     def forward(self, input, rois):
+        if torch._C._get_tracing_state():  # we cannot currently trace through the autograd function
+            return torch.ops.maskrcnn_benchmark.roi_align_forward(
```

Comment:
question: do we need to register the backwards for the op in C++ then so that we can perform training properly using the same codepath?

Reply:
The current obstacle here is that we cannot trace through the Python autograd function.
I'm not aware of a way to register the derivative so we could go through the op directly during training as well.

Reply:
Thanks! @goldsborough do you know if we can register derivatives for custom ops?

Reply:
Peter responded elsewhere, and "not quite yet", but it is a to-do on his list.